### DISCUSSION TITLE
Autogluon Tabular Prediction- Xgboost memory warning

### DISCUSSION THREAD

**Discussion by arminfar1** [autogluon/autogluon#2624] [Q&A] [answered]
I am using TabularPredictor for a multi-label prediction, and I keep getting this warning on xgboost. It is odd to me because the available memory s almost twice as the required memory. 
```
Warning: Potentially not enough memory to safely train XGBoost model, roughly requires: 6.29 GB, but only 17.844 GB is available...
```

**Comment by gradientsky** [comment, marked as answer]
It looks like the AutoGluon version used here is old (<0.4.2?), I suggest upgrading, we addressed this logic in #1626.
